app/src/main/java/pl/mdanilowski/spotted/data/model/City.kt

```diff
@@ -0,0 +1,5 @@
+package pl.mdanilowski.spotted.data.model
+
+import java.io.Serializable
+
+data class City(val name: String, val image: String?) : Serializable
```

Comment:
Parcelization is way faster than Serializable, look at Parcelizer: https://medium.com/@BladeCoder/a-study-of-the-parcelize-feature-from-kotlin-android-extensions-59a5adcd5909